app/views/camp/insert.php

```diff
@@ -0,0 +1,264 @@
+                                <div class="panel panel-primary">
+                                    <div class="panel-heading" style="font-size: large; padding-bottom: 15px">
+                                        <b>Camp Head Details</b>
+                                        <div class="btn btn-info" id="show1" style="float: right">Hide</div>
+                                    </div>
+                                    <div class="panel-body" id="panelBody1">
+                                        <label for="fname">First name </label>
+                                        <input id="fname" type="text" class="form-control" name="fname" placeholder="Enter First Name" required /><br>
+                                        <label for="lname">Last name </label>
+                                        <input type="text" class="form-control" placeholder="Enter Last Name" id="lname" name="lname" required /><br>
+                                        <label for="dob">Date of birth </label>
+                                        <input type="date" placeholder="Enter your DOB" class="form-control" id="dob" type="date" name="dob" /><br>
+                                        <label class="sr-only" for="gender">Gender: </label>
+                                        <div class="dropdown">
+                                            <select class="dropdown-header" id="gender" name="gender" required>
+                                                <option class="dropdown-toggle" value="" disabled selected>Select gender</option>
+                                                <option class="dropdown-header" value="F">Female</option>
+                                                <option class="dropdown-header" value="M">Male</option>
+                                                <option class="dropdown-header" value="O">Other</option>
+                                            </select><br>
+                                        </div>
+                                        <label for="ch_phone_no">Phone number </label>
+                                        <input id="ch_phone_no" class="form-control" placeholder="Enter Phone Number" name="ch_phone_no" required /><br>
+                                        <label for="ch_email">Email </label>
+                                        <input id="ch_email" placeholder="Enter Email" class="form-control" type="email" name="ch_email" required /><br>
+                                        <label for="ch_mailing_list">Mailing list </label>
+                                        <input id="ch_mailing_list" class="form-control" placeholder="Enter Mailing List" type="email" name="ch_mailing_list" /><br>
+                                        <label for="ch_mailing_address">Mailing address </label>
+                                        <input id="ch_mailing_address" class="form-control" placeholder="Enter Mailing Address" type="email" name="ch_mailing_address" /><br>
```

Comment:
This is not email address, this is physical address. 

Also it should be a textarea, not input.